src/saml2/entity_category/swamid.py

```diff
@@ -74,12 +106,26 @@
     (RESEARCH_AND_EDUCATION, NREN): NAME + STATIC_ORG_INFO + OTHER,
     (RESEARCH_AND_EDUCATION, HEI): NAME + STATIC_ORG_INFO + OTHER,
     RESEARCH_AND_SCHOLARSHIP: R_AND_S,
-    COCO: GEANT_COCO,
+    COCOv1: GEANT_COCO,
+    COCOv2: REFEDS_COCO,
     ESI: MYACADEMICID_ESI,
-    (ESI, COCO): MYACADEMICID_ESI + GEANT_COCO,
+    (ESI, COCOv1): MYACADEMICID_ESI + GEANT_COCO,
+    (ESI, COCOv2): MYACADEMICID_ESI + REFEDS_COCO,
+    # order is important for no aggregation categories as the last one wins if more than one is found
+    PERSONALIZED: REFEDS_PERSONALIZED_ACCESS,
+    PSEUDONYMOUS: REFEDS_PSEUDONYMOUS_ACCESS,
+    ANONYMOUS: REFEDS_ANONYMOUS_ACCESS,
 }
 
 ONLY_REQUIRED = {
-    COCO: True,
-    (ESI, COCO): True,
+    COCOv1: True,
+    COCOv2: True,
+    (ESI, COCOv1): True,
+    (ESI, COCOv2): True,
+}
+
+NO_AGGREGATION = {
+    PERSONALIZED: True,
```

Comment:
This is the set of the "no-aggregate" ECs. Is there a reason it is a map? I expect the value to be `True` always. I guess this should just be a set (or a list).

Reply:
The reason is just to mirror the ONLY_REQUIRED structure, do you want a set anyway?

Reply:
OK, let's leave it as a dict then. It allows us to be able to extend it later, if needed.